lib/ReportDataCollector.php

```diff
@@ -170,9 +179,18 @@ private function getBasicDetailArray() {
 				$homeCount++;
 			}
 		}
+		$licenseValidity = null;
+		foreach ($this->appConfigData as $index) {
+			foreach ($index as $key => $val) {
+				if ($key === 'license validity') {
+					$licenseValidity = $val;
+				}
+			}
+		}
 
 		return [
 			'license key' => $this->licenseKey,
+			'license validity' => $licenseValidity,
```

Comment:
no, this approach is wrong. We need extensions to be able to provide an array of keys and then add these keys (we never know their names in advance!) to the report.

Hardcoding license validity here is very wrong.